### ISSUE TITLE
Auto fix converts path in wrong format

### ISSUE THREAD

**Issue by Havunen** [joshuajaco/eslint-plugin-workspaces#1]
Hey!

Nice plugin, just what I thought I need to develop and then found this tool! :)

Ok, about this issue:
After running `eslint . --fix`

with following rule configuration `.eslintrc`: 

```
    // Workspaces
    "workspaces/no-absolute-imports": 2,
    "workspaces/no-cross-imports": 2,
    "workspaces/no-relative-imports": 2,
    "workspaces/require-dependency": 2
```

It converts following input (inside app/framework) workspace (rule: no-absolute-imports) :
```
import {SomeStore} from '@app/framework/src/somestore';
import {dispatcher} from '@app/framework/src/dispatcher';
```

to 

```
import {SomeStore} from '..\src\somestore';
import {dispatcher} from '..\src\dispatcher';
```

This is issue because backslash + s `\s` is handled as character `s`
and I get a lot of errors from other eslint rules:

```
ESLint: Unable to resolve path to module '..srcancestorstore'.(import/no-unresolved)
ESLint: Unnecessary escape character: \a.(no-useless-escape)
```

I think expected output of above code should be 

```
import {SomeStore} from '../src/somestore';
import {dispatcher} from '../src/dispatcher';
```

**Comment by Havunen** [comment]
I noticed 9 tests are failing when running tests in this repository using windows computer.

**Comment by joshuajaco** [comment]
Good find and fix 👍 Published in v0.5.1
